tests/interface/test_dialogs.py

```diff
@@ -57,7 +57,7 @@ def test_ModeSelector_setup():
     current_mode = 'python'
     mock_item = mock.MagicMock()
     with mock.patch('mu.interface.dialogs.ModeItem', mock_item):
-        ms = mu.interface.dialogs.ModeSelector()
+        ms = mu.interface.dialogs.ModeSelector(None)
```

Comment:
Why is `None` needed here..?

Also, CI complains because the tests don't pass. :-(

Reply:
With hindsight `__init__` should have `parent=None`